portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -13,17 +13,19 @@
 // limitations under the License.
 
 package com.google.sps.data;
-
-public class Comment //this object represents each comment posted to the website {Name: Comment}
+/*this object represents each comment posted to the website {Name: Comment} */
```

Comment:
Capitalize your comment and include the space in front.

`/* This object ...`

Reply:
I'll fix the comments